api-processor/src/main/java/com/synopsys/integration/alert/processor/api/extract/model/ProcessedProviderMessage.java

```diff
@@ -55,4 +59,13 @@ public T getProviderMessage() {
         throw new IllegalStateException("Combining models had more than two results");
     }
 
+    @Override
+    public int hashCode() {
+        return HashCodeBuilder.reflectionHashCode(17, 37, this, false, Stringable.class, "notificationIds");
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        return EqualsBuilder.reflectionEquals(this, obj, false, Stringable.class, true, "notificationIds");
```

Comment:
Same question as the HashCode method.
`EqualsBuilder.reflectionEquals(this, obj, EXCLUDED_COMPARISON_FIELDS);`

Reply:
Implemented the change to the simpler version.